src/main/java/org/terasology/oreGeneration/generation/OreRasterizer.java

```diff
@@ -42,16 +40,14 @@ public void initialize() {
     @Override
     public void generateChunk(CoreChunk chunk, Region chunkRegion) {
         OreFacet oreFacet = chunkRegion.getFacet(OreFacet.class);
-
-        BlockManager blockManager = CoreRegistry.get(BlockManager.class);
         OreGenRegistry oreGenRegistrySystem = CoreRegistry.get(OreGenRegistry.class);
 
         for (CustomOreGen oreGenCreator : oreGenRegistrySystem.iterateDefinitions()) {
             Map<StructureNodeType, Block> nodeTypeToBlocks = Maps.newHashMap();
-            for (Vector3i position : ChunkConstants.CHUNK_REGION) {
-                StructureNodeType nodeType = oreFacet.get(oreGenCreator, JomlUtil.from(position));
+            for (Vector3ic position : Chunks.CHUNK_REGION) {
+                StructureNodeType nodeType = oreFacet.get(oreGenCreator, position);
                 if (nodeType != null
-                    && oreGenCreator.canReplaceBlock(chunk.chunkToWorldPosition(JomlUtil.from(position), new org.joml.Vector3i()), chunkRegion)
+                    && oreGenCreator.canReplaceBlock(chunk.chunkToWorldPosition(position, new org.joml.Vector3i()), chunkRegion)
```

Comment:
We create a new vector here every time just to make a lookup. This looks like the perfect place to reuse a tmp vector instead.